Libplanet.Extensions.Cocona/Commands/StoreQueryCommand.cs

```diff
@@ -0,0 +1,135 @@
+        public void BlockByIndex(
+            [Argument("HOME", Description = "root home path")]
+            string home,
+            [Argument("BLOCK-INDEX", Description = "block index")]
+            int blockIndex
+        )
+        {
+            var store = new RocksDBStore.RocksDBStore(home);
+            if (!(store.GetCanonicalChainId() is { } chainId))
+            {
+                throw Utils.Error("Cannot find the main branch of the blockchain.");
+            }
+
+            if (!(store.IndexBlockHash(chainId, blockIndex) is { } blockHash))
+            {
+                throw Utils.Error(
+                    $"Cannot find the block with the height {blockIndex}" +
+                    $" within the blockchain {chainId}."
+                );
+            }
+
+            if (!(store.GetBlock<DummyAction>(blockHash) is { } block))
+            {
+                throw Utils.Error($"Cannot find the block with such hash: {blockHash}.");
+            }
+
+            Console.WriteLine(SerializeHumanReadable(block));
+        }
+
+        [Command(Description = "Query block by the hash")]
+        public void BlockByHash(
+            [Argument("HOME", Description = "root home path")]
+            string home,
+            [Argument("BLOCK-HASH", Description = "block hash")]
+            string blockHash
+        )
+        {
+            var store = new RocksDBStore.RocksDBStore(home);
+            if (!(store.GetBlock<DummyAction>(BlockHash.FromString(blockHash)) is { } block))
+            {
+                throw Utils.Error($"cannot find the block with the hash[{blockHash}]");
+            }
+
+            Console.WriteLine(SerializeHumanReadable(block));
+        }
+
+        [Command(Description = "Query transaction by the tx id")]
+        public void TxById(
+            [Argument("HOME", Description = "root home path")]
+            string home,
+            [Argument("TX-ID", Description = "tx id")]
+            string strTxId
+        )
+        {
+            var store = new RocksDBStore.RocksDBStore(home);
+            TxId txId = TxIdFromString(strTxId);
+            if (!(store.GetTransaction<DummyAction>(txId) is { } tx))
+            {
+                throw Utils.Error($"cannot find the tx with the tx id[{strTxId}]");
+            }
+
+            Console.WriteLine(SerializeHumanReadable(tx));
+        }
+
+        private static string SerializeHumanReadable<T>(T target) =>
+            JsonSerializer.Serialize(
+                target,
+                new JsonSerializerOptions
+                {
+                    WriteIndented = true,
+                    Converters =
+                    {
+                        new ByteArrayStringJsonConverter(),
+                    },
+                }
+            );
+
+        private static TxId TxIdFromString(string hex) =>
+            new TxId(ByteUtil.ParseHex(hex ?? throw new ArgumentNullException(nameof(hex))));
```

Comment:
It is not required to check when `hex` parameter is `null`. Because [the `Libplanet.Extensions.Cocona` project enables `Nullable` checker feature](https://github.com/planetarium/libplanet/blob/512c715522184580a0a221834575238fe910d3db/Libplanet.Extensions.Cocona/Libplanet.Extensions.Cocona.csproj#L17).

Reply:
removed the exception throwing.